src/models/HubbardAttractive/HubbardModelAttractive.jl

```diff
@@ -15,7 +15,10 @@ with linear system size `L`. Additional allowed `kwargs` are:
  * `U::Float64=1.0`: onsite interaction strength, "Hubbard U"
  * `t::Float64=1.0`: hopping energy
 """
-@with_kw_noshow mutable struct HubbardModelAttractive{C<:AbstractCubicLattice} <: Model
+@with_kw_noshow mutable struct HubbardModelAttractive{
+        LT <: Union{AbstractCubicLattice, LatPhysBase.AbstractLattice}
```

Comment:
I have to think about whether there is a smarter way than to use `Union` everywhere. Until then, it might make sense to define something like `const SomeLattice = Union{AbstractCubicLattice, LatPhysBase.AbstractLattice` once.

Reply:
More importantly, why allow a `LatPhysBase.AbstractLattice` here but only a `AbstractCubicLattice` in `hopping_matrix` below? Note that the former would work in `hopping_matrix` since we use `l.neighs` to access the neighbors their - a field which Jan's lattices don't have. We need a general solution here if we want to support all LatticePhysics lattices. Does Jan have a `getNeighbors` method?

Reply:
There are two definitions, one for `LatPhysBase.AbstractLattice` and one for `AbstractCubicLattice`. I'm using `bonds(::AbstractLattice)` instead of `l.neighs` there.

Reply:
Back to one hopping matrix as of 7387b0c, Union has been changed to `AbstractLattice` as of 3cbe4fa.